### PR TITLE
feat / updates to blocktane trading pair parser

### PR DESCRIPTION
**Pull request by trkoneill18**
**Before submitting this PR, please make sure**:

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

**Tests performed by the developer**:

**Tips for QA testing**:

### CODE REVIEW

**Comment by dennisocana** [comment]
Thank you for this update @trkoneill18 , please also address the flake8 error reported below.

image

**Comment by dennisocana** [comment]
@CoinAlpha/qa-team please test the proposed changes to Blocktane connector.

**Review by rxlxrxsx — APPROVED** [review]
Tested and confirmed working by QA.

File: hummingbot/connector/exchange/blocktane/blocktane_utils.py [hunk 1]
```diff
@@ -26,29 +27,57 @@
...
+    global INVERSE_MARKET_LOOKUP
+    global NAME_LOOKUP
+    if MARKET_DATA is None:
+        try:
+            markets = requests.get('https://trade.blocktane.io/api/v2/xt/public/markets').json()
+            MARKET_DATA = {market['id']: market for market in markets}
+            INVERSE_MARKET_LOOKUP = {(market['base_unit'].upper(), market['quote_unit'].upper()): market for market in markets}
+            NAME_LOOKUP = {market['name']: market for market in markets}
+        except Exception:
+            pass  # Will fall back to regex splitting
```
Comment (petioptrv) [inline hummingbot/connector/exchange/blocktane/blocktane_utils.py:47]: I think this should be a separate private function (e.g. `_populate_lookups`), called from both `split_trading_pair` and `convert_to_exchange_trading_pair`

**Review by petioptrv — CHANGES_REQUESTED** [review]
Very nice. Just one comment that I left in the code

**Review by petioptrv — APPROVED** [review]
Changes LGTM!

**Pull request merged**